pkg/virt-handler/virtwrap/errors.go

```diff
@@ -0,0 +1,47 @@
+/*
+ * This file is part of the kubevirt project
```

Comment:
KubeVirt should be capitalized here (and in all files where it's wrong, but that's outside of this PR)